tests/onnxruntime/test_modeling.py

```diff
@@ -144,7 +144,7 @@ def __init__(self, *args, **kwargs):
         self.ONNX_SEQ2SEQ_MODEL_ID = "optimum/t5-small"
         self.LARGE_ONNX_SEQ2SEQ_MODEL_ID = "facebook/mbart-large-en-ro"
         self.TINY_ONNX_SEQ2SEQ_MODEL_ID = "fxmarty/sshleifer-tiny-mbart-onnx"
-        self.TINY_ONNX_STABLE_DIFFUSION_MODEL_ID = "hf-internal-testing/tiny-random-OnnxStableDiffusionPipeline"
+        self.TINY_ONNX_STABLE_DIFFUSION_MODEL_ID = "IlyasMoutawwakil/tiny-stable-diffusion-onnx"
```

Comment:
this is because this pipeline doesn't even have configs, it used to load but sisnce inferencing with it wasn't tested, it didn't raise any issues, but there is an issue, which is that configs are needed for inference.

Reply:
can you move this model to https://huggingface.co/optimum-internal-testing (we try to progressively move our model for testing there like done for https://huggingface.co/optimum-internal-testing/tiny-random-llama)